docs/OtherAPIs.md

```diff
@@ -74,6 +74,21 @@ try {
 }
 ```
 
+## MIDDLEWAREOPTION
+
+You can pass in the middleware options for a request through `.middlewareOption()`. This takes array of strongly typed middleware options, these middleware options should be an implementation of MiddlewareOption interface
```

Comment:
Since many of our devs don't know English as a first language, our documentation should be as explicit as possible.

You can override the client middleware behavior by setting per request middleware options. Use the `.middlewareOption()` request builder method to add custom middleware behavior for a specific request. The `middlewareOption()` method takes an array of strongly typed middleware options.  These middleware options are an implementation of the [MiddlewareOption](../src/middleware/option/IMiddlewareOption.ts) interface.

Reply:
Should this method not be `middlewareOptions` considering it takes an array of options?

Reply:
Actually the method is named `middlewareOptions` but I have documented as `middlewareOption`. will change this.